src/components/folders/FolderItem.jsx

```diff
@@ -1,125 +1,27 @@
-import {
-  chakra,
-  LinkOverlay,
-  LinkBox,
-  Divider,
-  HStack,
-  Icon,
-  Text,
-  Box,
-} from "@chakra-ui/react"
-import { ContextMenu } from "components/ContextMenu"
-import { useMemo } from "react"
-import {
-  BiChevronRight,
-  BiEditAlt,
-  BiFolder,
-  BiTrash,
-  BiWrench,
-} from "react-icons/bi"
-import { Link as RouterLink, useRouteMatch } from "react-router-dom"
+import { Box } from "@chakra-ui/react"
 
 import elementStyles from "styles/isomer-cms/Elements.module.scss"
 import contentStyles from "styles/isomer-cms/pages/Content.module.scss"
 
 import { pageFileNameToTitle } from "utils"
 
-const FolderItem = ({ item, isDisabled }) => {
-  const { url } = useRouteMatch()
-
-  const { name, type, children } = item
-  const encodedName = encodeURIComponent(name)
-
-  const generatedLink = useMemo(() => {
```

Comment:
Just to clarify here - are we aware why there was a `useMemo` here is the first place?

Reply:
the `useMemo` is there to avoid recomputation of the `generatedLink` when stuff doesn't change. however, this component is only used in 1 place now (`ReorderingModal`) and will never be used as a `link` so this was removed